version.txt

```diff
@@ -1 +1 @@
-0.8.0-SNAPSHOT
+0.9.0-SNAPSHOT
```

Comment:
the main branch should go 0.9.0-SNAPSHOT now, and the branch-0.8 should go 0.8.1-SNAPSHOT now